RevenueCatUI/CustomerCenter/ViewModels/CustomerCenterViewModel.swift

```diff
@@ -266,19 +294,20 @@ private extension CustomerCenterViewModel {
                 return date1 < date2
             }) {
 
-            let entitlement = customerInfo.entitlements.all.values
-                .first(where: { $0.productIdentifier == subscription.productIdentifier })
-
             let purchaseInfo = await createPurchaseInformation(
                 for: subscription,
-                entitlement: entitlement,
                 customerInfo: customerInfo
             )
 
             activeSubscriptionPurchases.append(purchaseInfo)
         }
 
-        self.activeSubscriptionPurchases = activeSubscriptionPurchases
+        if activeSubscriptionPurchases.isEmpty {
+            // load last expired
```

Comment:
is it last or first? because the function says `first`. I assume it would be the last expired one? in that case we might want to rename the function

Reply:
Just pushed it